keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -23,8 +24,21 @@
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
 class RandomShear(BaseImageAugmentationLayer):
-    """Randomly shears an image.
-
+    """A preprocessing layer which randomly shears images during training.
+    This layer will apply random shearings to each image, filling empty space
+    according to `fill_mode`.
+    By default, random shears are only applied during training.
+    At inference time, the layer does nothing. If you need to apply random
+    shear at inference time, set `training` to True when calling the layer.
+    Input pixel values can be of any range (e.g. `[0., 1.)` or `[0, 255]`) and
+    of interger or floating point dtype. By default, the layer will output
```

Comment:
you can remove the return type information, maybe even all of the type information